src/main/java/com/example/videotecha/VideotechaApplication.java

```diff
@@ -1,11 +1,17 @@
 package com.example.videotecha;
 
+import com.example.videotecha.model.RegisteredUser;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.util.HashMap;
+import java.util.UUID;
+
 @SpringBootApplication
 public class VideotechaApplication {
 
+	public static HashMap<Long, RegisteredUser> users = new HashMap<>();
```

Comment:
Why is this defined here? Application is usually reserved for project-wide setups, this looks like a service variable